Documentation/Internationalization/Introduction/Index.rst

```diff
@@ -39,7 +39,7 @@ Here is a sample XLIFF file:
 .. code-block:: xml
 
    <?xml version="1.0" encoding="UTF-8"?>
-   <xliff version="1.0" xmlns:t3="http://typo3.org/schemas/xliff">
+   <xliff version="1.0" xmlns="urn:oasis:names:tc:xliff:document:1.2">
```

Comment:
We are using XLIFF v1.0, not 1.2 but since there is - AFAIK - no namespace definition so the namespace should read `document:1.1` at the end.